client/src/js/nav/components/Sidebar.js

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import { LinkContainer } from "react-router-bootstrap";
+import { Nav, NavItem } from "react-bootstrap";
+import { Icon } from "../../base";
+import { startsWith } from "lodash-es";
+
+const isHomeActive = (location) => {
+
+    let activeTab;
+
+    if (startsWith(location.pathname, "/jobs")) {
+        activeTab = "Jobs";
+    } else if (startsWith(location.pathname, "/samples")) {
+        activeTab = "Samples";
+    } else if (startsWith(location.pathname, "/viruses")) {
+        activeTab = "Viruses";
+    } else if (startsWith(location.pathname, "/subtraction")) {
+        activeTab = "Subtraction";
+    } else if (startsWith(location.pathname, "/administration")) {
+        activeTab = "Administration";
+    } else if (startsWith(location.pathname, "/hmm")) {
+        activeTab = "HMM";
+    }
+
+    return activeTab;
+};
+
+const Sidebar = (props) => {
+
+    const activeTab = isHomeActive(props.location);
+    let sidebarComponents;
+
+    switch (activeTab) {
+
+        case "Jobs":
```

Comment:
In this `switch`, you can directly return the component for each case instead of assigning it to `sidebarComponents` and returning at the end of the statement. This will also make `break` statements unnecessary.

The sidebar items are perfect candidates for being made into a reusable component.